packages/aws-cdk-lib/aws-ecs/README.md

```diff
@@ -610,6 +610,37 @@ const service = new ecs.ExternalService(this, 'Service', {
 `Services` by default will create a security group if not provided.
 If you'd like to specify which security groups to use you can override the `securityGroups` property.
 
+By default, the service will use the revision of the passed task definition generated when the `TaskDefinition`
+is deployed by CloudFormation. In order to specify a specific revision, pass a `taskDefinitionRevision`:
+
+```ts
+declare const cluster: ecs.Cluster;
+declare const taskDefinition: ecs.TaskDefinition;
+
+const service = new ecs.ExternalService(this, 'Service', {
+  cluster,
+  taskDefinition,
+  desiredCount: 5,
+  taskDefinitionRevision: '1'
+});
+```
+
+Or, to always use the latest active revision (for example, when using the CodePipeline EcsDeployAction
+without using the CODE_DEPLOY deployment controller to ensure future service deployments don't revert
+the task revision used by the service):
+
+```ts
+declare const cluster: ecs.Cluster;
+declare const taskDefinition: ecs.TaskDefinition;
+
+const service = new ecs.ExternalService(this, 'Service', {
+  cluster,
+  taskDefinition,
+  desiredCount: 5,
+  taskDefinitionRevision: 'latest'
+});
```

Comment:
To make sure I understand correctly, you think the description of the `latest` special case and why it would be used is unnecessary/too much information?

Reply:
No, but I think that documentation needs to be clear and direct.
Feel free to add examples or use cases to my suggestion.